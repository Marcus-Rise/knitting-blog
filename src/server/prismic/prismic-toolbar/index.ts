export { PrismicToolbar } from "./prismic-toolbar.component";
