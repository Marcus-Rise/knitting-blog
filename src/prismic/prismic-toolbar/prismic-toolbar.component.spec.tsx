import { render } from "@testing-library/react";
import { PrismicToolbar } from "./prismic-toolbar.component";
import React from "react";

jest.mock("next/head", () => ({
  __esModule: true,
  default: jest.fn((props) => <div data-testid="head">{props.children}</div>),
}));

describe("PrismicToolbar", () => {
  test("render", () => {
    const { asFragment } = render(<PrismicToolbar repositoryName={"repo"} />);

    expect(asFragment()).toMatchSnapshot();
  });
});
