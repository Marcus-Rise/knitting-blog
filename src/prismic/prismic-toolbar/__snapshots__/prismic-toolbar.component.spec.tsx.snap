// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PrismicToolbar render 1`] = `
<DocumentFragment>
  <div
    data-testid="head"
  >
    <script
      async=""
      defer=""
      src="https://static.cdn.prismic.io/prismic.js?new=true&repo=repo"
    />
  </div>
</DocumentFragment>
`;
